[#65] Used the suggested getOrAwaitValue() extension in the AboutViewModelTest tests

johnGachihi opened the pull request:
I have added and used the suggested `getOrAwaitValue()` extension in the `AboutViewModelTest` tests. I have also removed the `runBlockingTest` coroutine builder from the tests and the optional scope parameter from the `AboutViewModel` functions

johnGachihi committed:
[#65] Use getOrAwaitValue extension and remove runBlockingTest in AboutViewModelTest tests

- Add and use the suggested getOrAwaitValue() LiveData extension in
  AboutViewModelTest tests.
- Remove the runBlockingTest coroutine builders since they are not required
  any more

johnGachihi committed:
[#65] Remove now unnecessary optional scope parameter

wangerekaharun commented:
@michaelbukachi can you have a look on this?

michaelbukachi commented:
> @michaelbukachi can you have a look on this?

I'm waiting for the commit that integrates `TestCoroutineDispatcher` as mentioned in #65

johnGachihi committed:
[#65] Change main coroutine dispatcher set by `CoroutinesRule` to be TestCoroutineDispatcher

johnGachihi committed:
[#65] Delete the getOrWaitValue LiveData extension

Delete the getOrWaitValue LiveData extension function and replace
it in `AboutViewModelTest` with the usual `LiveData.value`

johnGachihi commented:
@michaelbukachi Sorry for the delay. I misunderstood your comment.
